ckanext/schema/dc_dataset.yaml

```diff
@@ -0,0 +1,321 @@
+  # On automated submissions, README.md file will populate this field if present
+  - field_name: notes
+    label: Description
+    required: true
+    preset: markdown
+    display_property: dct:description
+    help_text: A Detailed description of the dataset, including data quality and processing techniques.
+
+  # Tags will come from preset list
+  # TODO determine relevant climate-themed tag list
+  # TODO create custom preset that serves predefined tag list
+  - field_name: tag_string
+    label: Keywords
+    required: false
+    preset: tag_string_autocomplete
+    display_property: dcat:keyword
+    form_placeholder: eg. electricity, policy, land use
+
+  # Will be used to build a filter (e.g. by year)
+  - field_name: temporal_start
+    label: Start date
+    required: false
+    preset: date
+    display_property: dcat:startDate
+    help_text: Earliest date the dataset covers e.g. 2016-01-01.
+    
+  # Will be used to build a filter (e.g. by year)
+  - field_name: temporal_end
+    label: End date
+    required: false
+    preset: date
+    display_property: dcat:endDate
+    help_text: Latest date the dataset covers e.g. 2019-01. Leave blank if dataset is updated to present day.
+
+  # Will be used to build a filter
+  - field_name: temporal_resolution
+    label: Time resolution
+    required: false
+    preset: select
+    choices:
+    - value: sub_second
+      label: Sub-second
+    - value: second
+      label: Second
+    - value: minute
+      label: Minute
+    - value: hour
+      label: Hour
+    - value: day
+      label: Day
+    - value: week
+      label: Week
+    - value: month
+      label: Month
+    - value: quarter
+      label: Quarter
+    - value: year
+      label: Year
+    display_property: dcat:temporalResolution
+    help_text: For time-series data, please select the smallest resolution represented in your dataset. If your dataset resolution falls between the values given here, please select the closest resolution (e.g. for readings every 30 seconds select minute).
+  
+  # Could be used to automate communication with publisher to update their dataset
+  # Could also feed into freshness or future data quality scores
+  - field_name: update_frequency
+    label: Update frequency
+    required: false
+    preset: select
+    choices:
+    - value: hourly
+      label: Hourly
+    - value: daily
+      label: Daily
+    - value: weekly
+      label: Weekly
+    - value: monthly
+      label: Monthly
+    - value: quarterly
+      label: Quarterly
+    - value: annually
+      label: Annually
+    - value: never
+      label: Never
+    display_property: dcat:accrualPeriodicity
+    help_text: Select the closest value that reflects the frequency with which the dataset is updated.
+
+  # Will be used to build a filter
+  - field_name: iso_region
+    label: ISO region
+    required: false
+    preset: select
+    choices:
```

Comment:
just been looking into this and can't see ISO codes for regions, just countries. We might have to use our own sensible list. 
World, North America, South America, Africa, Asia, South East Asia, Australasia, Europe
?
can get tricksy quickly though

Reply:
Perhaps the top-level entries in the M49 standard: https://unstats.un.org/unsd/methodology/m49/. So that would be:
- World
- Africa
- Americas
- Asia
- Europe
- Oceania